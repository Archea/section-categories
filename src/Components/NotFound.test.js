import React from 'react'
import ReactDOM from 'react-dom'
import { NotFound } from './NotFound'
import renderer from 'react-test-renderer'

describe('NotFound', () => {
  test('it at renders and matches snapshot', () => {
    const comp = renderer.create(<NotFound />)
    expect(comp).toBeDefined()
    expect(comp.toJSON()).toMatchSnapshot()
  })
})
