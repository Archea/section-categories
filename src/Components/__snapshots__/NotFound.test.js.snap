// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound it at renders and matches snapshot 1`] = `
<div>
  <div>
    Page Not Found
  </div>
</div>
`;
